.github/workflows/docker-hub-release.yml

```diff
@@ -18,6 +18,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Get release tag
         run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set up Docker Buildx
```

Comment:
I *think* you also need this bit https://github.com/matrix-org/matrix-appservice-slack/blob/develop/.github/workflows/docker-hub-release.yml#L27-L28